src/mail/view/EventBanner.ts

```diff
@@ -137,3 +136,12 @@ function sendResponse(event: CalendarEvent, recipient: string, status: CalendarA
 		}),
 	)
 }
+
+/**
+ * get the attendee from the event that corresponds to the given recipient address, if there is one.
+ */
+function getOwnAttendee(event: CalendarEvent, recipient: string): CalendarEventAttendee | null {
```

Comment:
I'd name it `findAttendee`, also we should probably put it somewhere common because `CalendarEventViewModel` also has `findOwnAttendee` which is broken in the same exact way I think?